src/drivers/git.ts

```diff
@@ -0,0 +1,84 @@
+import fs, { existsSync, promises as fsp } from 'fs'
+import git from 'isomorphic-git'
+import http from 'isomorphic-git/http/node'
```

Comment:
We could create an additional driver like `git.node` for node-imports. By default I would rather to use isomorphic (web) implementation